back/engines/commercial/user_custom_fields/app/controllers/user_custom_fields/web_api/v1/stats_users_controller.rb

```diff
@@ -106,7 +106,11 @@ def user_counts_by_area_id
 
         def find_users
           users = policy_scope(User.active, policy_scope_class: StatUserPolicy::Scope)
-          finder_params = params.permit(:group, :project).merge(registration_date_range: @start_at..@end_at)
+          finder_params = if params[:filter_by_participation]
+            params.permit(:group, :project, :filter_by_participation).merge({ participation_date_range: { since: @start_at, to: @end_at } })
```

Comment:
Do you need to permit `:filter_by_participation`? Is it used further? If not, it would be enough
```suggestion
            params.permit(:group, :project).merge({ participation_date_range: { since: @start_at, to: @end_at } })
```

Reply:
yes, it works the same, thanks!